src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheck.java

```diff
@@ -279,6 +278,17 @@ else if (!allowNoEmptyLineBetweenFields
     }
 
     /**
+     * Checks if a token has empty two previous lines and there is not allowed multiple empty lines
```

Comment:
Change `there is not allowed multiple empty lines` to `multiple empty lines are not allowed`